main.go

```diff
@@ -117,7 +119,7 @@ func validateDeps(deps []depEntry) error {
 		if len(rootDeps) == 1 {
 			d := rootDeps[0]
 			if d.importPath != r {
-				log.Printf("WARNING: package %s is not root import, should be %s", d.importPath, r)
```

Comment:
Yes, but I think it is good to use `Warnf` consistently across all the non-trivial warnings